kolibri/auth/api.py

```diff
@@ -233,21 +233,24 @@ def get_session(self, request):
                    'user_id': user.id,
                    'can_manage_content': user.can_manage_content}
         roles = Role.objects.filter(user_id=user.id)
-        if len(roles) is not 0 or user.is_superuser:
+
+
+        if len(roles) is 0:
```

Comment:
Ideally we should not be mixing superuser (permission type) with role type. They should be separate. But for now the change I made is that superuser learners also have the learner role.